README.md

```diff
@@ -1 +1,30 @@
-Easypay Checkout SDK
+# Easypay Checkout SDK
+
+## To install dependencies:
+
+    yarn install
+
+## To build:
+
+    yarn build
+
+## To publish:
+
+### Public access:
+
+    npm publish --access public
+
+or
+Add to _**package.json**_:
+
+    "publishConfig": {
+        "access": "public"
+    },
+
+and then use:
+
+    npm publish
+
+### Private access:
+
+    npm publish
```

Comment:
`package.json` is already in the repo, so I would remove this entire complication altogether and just leave the relevant instruction.

```suggestion
    npm publish
```

Reply:
"this entire complication altogether" ? 🤔 
Ok will do a entire simplification altogether 😂